playbooks/files

```diff
@@ -0,0 +1 @@
+../../../../files
```

Comment:
Why adding this symlink ? We do not have it in other collection.

Reply:
it was generated by the setup.sh. I will remove it from the commit.